js/utils.js

```diff
@@ -104,7 +98,7 @@ function getSelectedParticipantUrls(participants) {
   return urls;
 }
 
-async function fetchParticipantWebIDs(fetch, employeesUrl) {
+export async function fetchParticipantWebIDs(fetch, employeesUrl) {
```

Comment:
I fixed the issue on my side by passing the parameter `participants` do this function. I guess that with making it webpack it doesn't these global variables anymore.